stdlib/LinearAlgebra/src/symmetriceigen.jl

```diff
@@ -184,6 +184,50 @@ function eigen!(A::AbstractMatrix, C::Cholesky; sortby::Union{Function,Nothing}=
     GeneralizedEigen(sorteig!(vals, vecs, sortby)...)
 end
 
+# Bunch-Kaufmann (LDLT) based solution for generalized eigenvalues and eigenvectors
+function eigen(A::AbstractMatrix{T}, B::BunchKaufman{T,<:StridedMatrix}; sortby::Union{Function,Nothing}=nothing) where {T<:Number}
+    eigen!(copy(A), copy(B); sortby)
+end
+function eigen!(A::AbstractMatrix{T}, B::BunchKaufman{T,<:StridedMatrix}; sortby::Union{Function,Nothing}=nothing) where {T<:Number}
+   # NOTE: getproperty(B, :D) and getproperty(B, :L/U) are not in-place. Hence, in the following, sysconvf/_rook is
+   # directly utilized to obtain M, du, d = diag(LUD), and dl. See bunchkaufman.jl/getproperty for details.
+   if B.rook
+        LUD, od = LAPACK.syconvf_rook!(B.uplo, 'C', B.LD, B.ipiv)
+    else
+        LUD, od = LAPACK.syconv!(B.uplo, B.LD, B.ipiv)
+    end
+    if B.uplo == 'U'
+        M = UnitUpperTriangular(LUD)
+        du = od[2:end]
+        # Aliasing of dl and du is not allowed for gtsv! below.
+        dl = B.symmetric ? copy(du) : conj.(copy(du))
```

Comment:
Applying broadcasting to a vector, even if it's `identity`, yields an unaliased result.
```suggestion
        dl = B.symmetric ? copy(du) : conj.(du)
```
Similarly below.

Reply:
Thank you for this comment and the information. As you're aware, I've only started using Julia seriously since a few months, and information such as this is very helpful. I've made this change as well as a part of https://github.com/aravindh-krishnamoorthy/julia/commit/8b8f556d7095452e14a3bc7c46c4bc2f34d6bb9d.